00-HTML-CSS-basics/Exercise 4.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Exercise v</title>
+<meta charset="UTF-8">
+<meta name="description" content="Forms">
+<meta name="keywords" content="HTML,BOOTCAMP">
+<meta name="author" content="Octavio Bruzzesi">
+<meta name="viewport" content="width=device-width,initial-scale=1">
+</head>
+<body>
+
+<h1>Sign up</h1>
+
+<form>
+   First name:<br>
```

Comment:
It is a good practice to add a label with the input. Check an example here http://www.w3schools.com/tags/tag_label.asp